samples/sdl_gamecontroller/main.c

```diff
@@ -0,0 +1,46 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  SDL_GameController *gameController;
+  gameController = SDL_GameControllerOpen(0);
```

Comment:
nit: can be `SDL_GameController *gameController = SDL_GameControllerOpen(0);`

*Existing code is already like this: `int ret = pb_init();`*